### DISCUSSION TITLE
README missing Criteria info

### DISCUSSION THREAD

**Discussion by warp9pnt9** [ScoopInstaller/Extras#11588] [General]
The README.md refers to the criteria for Main.  That document describes what belongs in Main.  All Extras says is "anything that doesn't belong in Main".  Then I guess the world only needs 2 buckets?  What about all the other buckets?  I can't find criteria info in them.  The project needs to better communicate the specific criteria for each bucket, in an obvious place like the README.  Make no assumptions.  We are not all psychics. E.g. what is the distinction between Extras and Non-Portable?  Examples of some of the most popular software in the bucket, etc.  I can't find a package after searching many repos.  So I want to try to add it myself.  But then, which Bucket?  It's really not clear, and requires asking, which wastes everyone's time and energy due to failure to communicate effectively in the README.md files.

**Comment by HUMORCE** [comment]
> Criteria
> For a package to be acceptable in this bucket, it should be
> 
>  - Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 100 stars and/or 50 forks) *
>  - English interface (or at least English documentation) *
>  - Latest stable version *
>  - Full version (i.e. not a trial version)
>  - Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

You can see criteria of Extras bucket when you create a package request issue or open an existing package request issue.

---

> Then I guess the world only needs 2 buckets?

Yes, you could think so. Buckets except Main, Extras, Versions and Nonportable are used to categorize.

Versions are used to archive/add specific versions of apps.

Nonportable is a special bucket that is used to add manifests that do not meet [Scoop's criteria](https://github.com/ScoopInstaller/Scoop#what-does-scoop-do).

---

I don't think you're going to add game to Extras bucket or JDK/JRE distribution to Main bucket after seeing [this](https://github.com/ScoopInstaller/Scoop#known-application-buckets):

> Known application buckets
> - [main](https://github.com/ScoopInstaller/Main) - Default bucket for the most common (mostly CLI) apps
> - [extras](https://github.com/ScoopInstaller/Extras) - Apps that don't fit the main bucket's [criteria](https://github.com/ScoopInstaller/Scoop/wiki/Criteria-for-including-apps-in-the-main-bucket)
> - [games](https://github.com/Calinou/scoop-games) - Open source/freeware games and game-related tools
> - [nerd-fonts](https://github.com/matthewjberger/scoop-nerd-fonts) - Nerd Fonts
> - [nirsoft](https://github.com/kodybrown/scoop-nirsoft) - Almost all of the [250+](https://rasa.github.io/scoop-directory/by-apps#kodybrown_scoop-nirsoft) apps from [Nirsoft](https://nirsoft.net/)
> - [sysinternals](https://github.com/niheaven/scoop-sysinternals) - Sysinternals Suite and all individual application from [Microsoft](https://learn.microsoft.com/sysinternals/)
> - [java](https://github.com/ScoopInstaller/Java) - A collection of Java development kits (JDKs), Java runtime engines (JREs), Java's virtual machine debugging tools and Java based runtime engines.
> - [nonportable](https://github.com/ScoopInstaller/Nonportable) - Non-portable apps (may require UAC)
> - [php](https://github.com/ScoopInstaller/PHP) - Installers for most [versions](https://github.com/ScoopInstaller/Versions) of PHP
> - [versions](https://github.com/ScoopInstaller/Versions) - Alternative versions of apps found in other buckets

---

If you have suggetions for bucket's docs, contributions welcome.
